### ISSUE TITLE
[Bug] Aliases doesn't work

### ISSUE THREAD

**Issue by gamerover98** [Despical/CommandFramework#3]
## Description of Issue
Hi, I'm trying this library and I have found the first hiccup. By following exactly the wiki, the `example` command works but aliases `firstAlias` and `secondAlias` don't. Typing the aliases from the console or from the chat the result is the same: `Unknown command. Type "/help" for help`.

## Steps to Reproduce
Use the wiki [example usage](https://github.com/Despical/CommandFramework/wiki/Command-examples#creating-sub-command).

## Command Framework Version
1.2.0

## Server Version
Tested on: Spigot 1.8.8 and Spigot 1.20.1

## Stack Trace/Log Files/Crash Reports
```LOG
> firstAlias
[12:49:34 INFO]: Unknown command. Type "/help" for help.
> secondAlias
[12:49:34 INFO]: Unknown command. Type "/help" for help.
```

## Additional Information
No additional information.

----------

Looking forward to your feedback and collaboration on this enhancement.
Thank you

**Comment by Despical** [comment]
I'll check in a few hours, thanks for the report.

**Comment by Despical** [comment]
Bukkit does not support case-sensitive command aliases if any alias contains upper letter and the issue is fixed in [5b85d9e](https://github.com/Despical/CommandFramework/commit/5b85d9ed6b2f896a208ccf8c4ed0684ccf5e4939).
